caikit/runtime/model_management/model_loader.py

```diff
@@ -96,14 +99,26 @@ def load_model(
         return model_builder.build()
 
     def _load_module(
-        self, model_path: str, model_id: str, model_type: str
+        self,
+        model_path: str,
+        model_id: str,
+        model_type: str,
+        finder: Union[None, str, ModelFinderBase],
+        initializer: Union[None, str, ModelInitializerBase],
```

Comment:
I think these semantics would be a little cleaner for the type hint (same as above)

```suggestion
        finder: Optional[Union[str, ModelFinderBase]],
        initializer: Optional[Union[str, ModelInitializerBase]],
```

Reply:
Good call! I also updated them to be kwargs